modules/i3s/package.json

```diff
@@ -47,9 +47,12 @@
     "@loaders.gl/draco": "4.2.0-alpha.5",
     "@loaders.gl/images": "4.2.0-alpha.5",
     "@loaders.gl/loader-utils": "4.2.0-alpha.5",
+    "@loaders.gl/math": "4.2.0-alpha.5",
+    "@loaders.gl/polyfills": "4.2.0-alpha.5",
```

Comment:
Is `@loaders.gl/polyfills` meant to be in `devDependencies`, `dependencies`, or both?  Usually in a monorepo the subpackages do not have `devDependencies`, so I'm not sure whether this ...

https://github.com/search?q=repo%3Avisgl%2Floaders.gl+%40loaders.gl%2Fpolyfills+path%3Apackage.json&type=code

... is an oversight, or has a purpose I'm not aware of.

Reply:
Polyfills should be included by client app under node.js only. No loaders module should depend on it. The module could be renamed to "node" at some point.

Reply:
Ok, thanks! I've removed `@loaders.gl/polyfills` from every loaders module _except_ for tile-converter, which appears to be an exception — it includes a CLI executable that depends on the polyfills.